pennylane/operation.py

```diff
@@ -1132,6 +1131,28 @@ def arithmetic_depth(self) -> int:
         """Arithmetic depth of the operator."""
         return 0
 
+    def map_wires(self, wire_map: dict):
+        """Returns a copy of the current operator with its wires changed according to the given
+        wire map.
+
+        Args:
+            wire_map (dict): dictionary containing the old wires as keys and the new wires as values
+
+        Returns:
+            .Operator: new operator
+        """
+        if any(wire not in wire_map for wire in self.wires):
```

Comment:
If the wire isn't in the wire map, we could just leave it unchanged.  

```
>>> qml.CNOT((0,1)).map_wires({1: "a"})
CNOT(wires=(0,"a"))
```

I think that behaviour makes sense personally. Allows us to define an error out of existence.

Reply:
Makes sense!

Reply:
Changed